### ISSUE TITLE
[QUERY]Is the Xamarin.Forms nuget package is available to public?

### ISSUE THREAD

**Issue by jessejiang0214** [Azure/azure-notificationhubs-xamarin#12]
**Query/Question**
Just want to confirm where can I find the Nuget package for the Xamarin.Form project?

***Why is this not a Bug or a feature Request?***
I want to use the new API in my project, but I cannot find the nuget.

**Setup (please complete the following information if applicable):**
 - OS: [e.g. iOS 13.x]
 - IDE : [e.g. Visual Studio 2019]
 - Version of the Library used
 
 **Information Checklist**
 Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [ ] Query Added
- [ ] Setup information Added

**Comment by mauro-dasilva** [comment]
Yeah, I was wondering the same thing. I cant seem to find it anywhere.

**Comment by adambarath** [comment]
It is not available on nuget. Documentation of this repository starts clearing it as well:

> This project contains Microsoft.Azure.NotificationHubs.Client, which when added to your project, provides a consistent experience across Xamarin Android and Xamarin for Apple. Either **download or clone the repository** and either **add the source** from the project to your own **solution,** _or_ **create a NuGet package from the source** and add from a local NuGet source.

**Comment by StephanRaab** [comment]
https://www.nuget.org/packages/Azure-NotificationHubs.Client.Xamarin/1.1.0.1-beta

**Comment by Phenek** [comment]
Hey great to found this package, because Microsoft documentation is outdated...

Is it possible to release this in official nuget package, please?
@mpodwysocki, @microsoftopensource 

Thx anyway for your work <3
